src/components/Input/Toggle/constants.js

```diff
@@ -0,0 +1,3 @@
+const TOGGLE_SIZES = ["small", "regular"];
```

Comment:
other inputs introduce sizes as "small" & "large" - we should keep it consistent.

Reply:
I got confused with names in design specs (it's small and regular there). Changed to small and large.